### DISCUSSION TITLE
Add comparison to liquibase

### DISCUSSION THREAD

**Discussion by asm0dey** [amacneil/dbmate#565] [Feature Ideas]
There are currently many tools to which dbmate is compared, but Liquibase is absent. It would be nice to see it in the comparison table too!

**Comment by amacneil** [comment]
Never heard of it - PRs welcome, although we are getting low on horizontal space. 

Liquibase looks like a saas service that wants me to register for a webinar, so at first glance it is probably not a relevant comparison.

**Comment by asm0dey** [comment]
No, it's totally a standalone command line tool. Contrary to flyway it can reverse transactions, for example :)
And supports different kinds of migrations - xml, SQL, etc
